src/zenml/zen_stores/migrations/versions/e5225281b4d3_add_connector_skew_tolerance.py

```diff
@@ -0,0 +1,35 @@
+"""add connector skew tolerance [e5225281b4d3].
+
+Revision ID: e5225281b4d3
+Revises: 0.51.0
+Create Date: 2023-12-08 13:50:25.321142
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "e5225281b4d3"
+down_revision = "0.51.0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade database schema and/or data, creating a new revision."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("service_connector", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("expires_skew_tolerance", sa.Integer(), nullable=True)
```

Comment:
Shall it be back-populated with some value?

Reply:
@avishniakov I modeled it as an optional attribute to make the migration easier. If not configured, the code will just use the default value of 5 minutes saved as a constant in the code.